README.md

```diff
@@ -4,6 +4,43 @@ Shimmer is a collection of Rails extensions that bring advanced UI features into
 
 ## Features
 
+### Components
+
+Shimmer includes a suite of styled components that can be implemented in React and also in plain HTML or slim.
+
+#### Stack
+
+Stack is a reusable typed component that allows you to easily manage the layout of your app. You can define whether it should be displayed horizontally, vertically, and how much spacing there should be in between the child components. This component implements a mobile-first design and allows you to customize the display and spacing even on defined breakpoints (tablet, desktop, widescreen) should you need to.
+
+To use it in a React project, you can just import and use it as you would in a normal React component:
+
+```js
+import { Stack } from '@nerdgeschoss/shimmer/components/stack';
+
+<Stack gapTablet={4} gapDesktop={12} line>
+  <div></div>
+  <div></div>
+  <div></div>
+</Stack>
+```
+
+To use it in an HTML file, you can just import the css file directly from `@nerdgeschoss/shimmer/components/stack.css` and just implement the classes as they are in the stylesheet:
```

Comment:
```suggestion
To use it in an HTML file, you can just import the css file directly from `@nerdgeschoss/shimmer/components/stack.scss` and just implement the classes as they are in the stylesheet:
```

Reply:
There's also a css export, right? @grcldq

Reply:
Once it's compiled it will be a .css file, not scss